### DISCUSSION TITLE
Api-News

### DISCUSSION THREAD

**Discussion by itsleviaa10** [deaafrizal/forum#74] [Troubleshooting Room] [answered]
Tanya dong kalo mau buat sebuah blog/artikel berita biasanya pakai api apa ya, dan nama website yang menyedikan api berita apa aja ya. Mohon bantuannya, saya masih belajar api jadi saya kurang tau web-web apa aja yang nyediain api.

**Comment by seporang** [comment, marked as answer]
kamu bisa pakai newsapi lalu pakek headline indonesia, btw itu cuman cari berita yang lagi populer

**Reply by itsleviaa10** [reply]
Makasih : )
